### PR TITLE
terraform/*: add German translation

### PR DESCRIPTION
**Pull request by jnmcfly**
translate terraform pages to german language

- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/terraform-plan.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`terraform plan -target {{resource_type.resource_name[instance index]}}`
+
+- Ausgabe eines Plans als JSON:
+
+`terraform plan -json`
+
+- Ausgabe eines Plans in eine separate Datei:
+
+`terraform plan -no-color > {{path/to/file}}`
```
Comment (kbdharun) [inline pages.de/common/terraform-plan.md:32]:
```suggestion
`terraform plan -no-color > {{pfad/zu/datei}}`
```

File: pages.de/common/terraform-plan.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`terraform plan -refresh-only`
+
+- Werte für Eingabevariablen festlegen:
+
+`terraform plan -var '{{name1}}={{value1}}' -var '{{name2}}={{value2}}'`
+
+- Anzeigen eines Plans auf eine Teilmenge von Ressourcen:
+
+`terraform plan -target {{resource_type.resource_name[instance index]}}`
```
Comment (kbdharun) [inline pages.de/common/terraform-plan.md:24]: Can you translate the command argument `{{resource_type.resource_name[instance index]}}`?

File: pages.de/common/terraform-plan.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`terraform plan -destroy`
+
+- Anzeigen eines Plans zur Aktualisierung des Terraform-Status und der Ausgabewerte:
+
+`terraform plan -refresh-only`
+
+- Werte für Eingabevariablen festlegen:
+
+`terraform plan -var '{{name1}}={{value1}}' -var '{{name2}}={{value2}}'`
```
Comment (kbdharun) [inline pages.de/common/terraform-plan.md:20]: Can you translate `name` and `value` here?

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr. Thanks for your contribution. I have some suggestions for this page.

> Note: Translate command arguments inside `{{}}` along with command descriptions in future.

File: pages.de/common/terraform-fmt.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# terraform fmt
+
+> Formatierung der Konfiguration gemäß den Konventionen der Terraform-Sprache.
+> Mehr Informationen: <https://www.terraform.io/docs/commands/fmt.html>.
```
Comment (kbdharun) [inline pages.de/common/terraform-fmt.md:4]:
```suggestion
> Weitere Informationen: <https://www.terraform.io/docs/commands/fmt.html>.
```

File: pages.de/common/terraform-plan.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# terraform plan
+
+> Erzeugen und Anzeigen von Terraform-Ausführungsplänen.
+> Mehr Informationen: <https://www.terraform.io/docs/cli/commands/plan.html>.
```
Comment (kbdharun) [inline pages.de/common/terraform-plan.md:4]:
```suggestion
> Weitere Informationen: <https://www.terraform.io/docs/cli/commands/plan.html>.
```

File: pages.de/common/terraform.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# terraform
+
+> Erstellen und Bereitstellen von Infrastruktur als Code für Cloud-Anbieter.
+> Mehr Informationen: <https://www.terraform.io/>.
```
Comment (kbdharun) [inline pages.de/common/terraform.md:4]:
```suggestion
> Weitere Informationen: <https://www.terraform.io/>.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**